ansible_runner/runner_config.py

```diff
@@ -568,21 +611,125 @@ def _ensure_path_safe_to_mount(path):
+                    elif _book_keeping_copy[_book_keeping_copy.index(arg - 1)][0] != '-':
+                        _playbook = arg
+                        break
+
+            return (_playbook, _inventory_paths)
+
+        if self.cli_execenv_cmd:
+            _parsed_playbook_path, _parsed_inventory_paths = _parse_cli_execenv_cmd_playbook_args()
+            if self.cli_execenv_cmd == 'playbook':
+                playbook_file_path = _parsed_playbook_path
+                _ensure_path_safe_to_mount(playbook_file_path)
+                if os.path.isabs(playbook_file_path) and (os.path.dirname(playbook_file_path) != '/'):
+                    new_args.extend([
+                        "-v", "{}:{}".format(
+                            os.path.dirname(playbook_file_path),
+                            os.path.dirname(playbook_file_path),
+                        )
+                    ])
+                else:
+                    new_args.extend([
+                        "-v", "{}:/runner/project/{}".format(
+                            os.path.dirname(os.path.abspath(playbook_file_path)),
+                            os.path.dirname(playbook_file_path),
+                        )
+                    ])
+
+            # volume mount inventory into the exec env container if provided at cli
+            if '-i' in self.cmdline_args:
+                inventory_file_paths = _parsed_inventory_paths
+                inventory_playbook_share_parent = False
+                for inventory_file_path in inventory_file_paths:
+                    _ensure_path_safe_to_mount(inventory_file_path)
+                    if self.cli_execenv_cmd == 'playbook':
+                        if os.path.dirname(os.path.abspath(inventory_file_path)) == \
+                                os.path.dirname(os.path.abspath(playbook_file_path)):
+                            inventory_playbook_share_parent = True
+                    if not inventory_file_path.endswith(',') and not inventory_playbook_share_parent:
+                        if os.path.isabs(inventory_file_path) and (os.path.dirname(inventory_file_path) != '/'):
+                            new_args.extend([
+                                "-v", "{}:{}".format(
+                                   os.path.dirname(inventory_file_path),
+                                   os.path.dirname(inventory_file_path),
+                                )
+                            ])
+                        else:
+                            new_args.extend([
+                                "-v", "{}:/runner/project/{}".format(
+                                   os.path.dirname(os.path.abspath(inventory_file_path)),
+                                   os.path.dirname(inventory_file_path),
+                                )
+                            ])
+
+            # volume mount ~/.ssh/ into the exec env container
+            new_args.extend(["-v", "{}/.ssh/:/runner/.ssh/".format(os.environ['HOME'])])
```

Comment:
Just noticed I broke `known_hosts` support when I changed the home dir location. This should fix it:

```suggestion
            new_args.extend(["-v", "{}/.ssh/:/home/runner/.ssh/".format(os.environ['HOME'])])
```